recipes-support/curl/curl_7.69%.bbappend

```diff
@@ -0,0 +1,6 @@
+# enable HTTP2 support in curl for AVS Device SDK
+DEPENDS_class-target_append = " nghttp2"
```

Comment:
Is this needed? Below you are defining the PACKAGECONFIG option to build the nghttp2 and add that to the list

Reply:
removed